src/Be.Vlaanderen.Basisregisters.AcmIdm/AuthorizationHandlers/AcmIdmAuthorizationHandler.cs

```diff
@@ -10,8 +10,8 @@
 
     public class AcmIdmAuthorizationHandler : AuthorizationHandler<AcmIdmAuthorizationRequirement>
     {
-        private static readonly ReadOnlyDictionary<string, string> _cache =
-            new ReadOnlyDictionary<string, string>(new Dictionary<string, string>(1000));
+        // Todo: is this type thread safe when making concurrent writes to the dictionary cache?
```

Comment:
doesn't matter, will be removed when we have a NisCode service

Reply:
Removed comment.